### PR TITLE
Experimental BlurView

### PR DESCRIPTION
**Pull request by brentvatne**
For discussion with @chirag04

Example usage:

```
  static route = {
    navigationBar: {
      renderTitle: () => <BrandedNavigationTitle />,
      translucent: true,
    },
  }
```

In `render`, in order to get the top offset..

```
  render() {
    const { route } = this.props;

    return (
      <ScrollView
        style={styles.container}
        contentContainerStyle={route.getContentContainerStyle()}>
      // etc..
```

### CODE REVIEW

File: src/ExNavigationBar.js [hunk 1]
```diff
@@ -14,6 +14,8 @@ import {
 } from 'react-native';
 import PureComponent from 'PureComponent';
 
+import { Components } from 'exponent';
```
Comment (chirag04) [inline src/ExNavigationBar.js]: not sure what the best way to make it work for non-exponent apps.

**Comment by brentvatne** [comment]
This has been merged

File: src/ExNavigationBar.js (hunk 1 above)
Comment (rogchap) [inline src/ExNavigationBar.js]: For non-exponent apps, would it make sense to pass-in a BlurView as a `prop`? Then you could just use: https://github.com/react-native-community/react-native-blur if you wanted.

Comment (rogchap) [inline src/ExNavigationBar.js]: My current solution is to hack away at my own `global.__exponent` 😨

Comment (tomtwo) [inline src/ExNavigationBar.js]: Could you please share your solution for this? In the same boat! Thanks.

**Pull request closed without merging**